telluric/georaster.py

```diff
@@ -395,6 +375,60 @@ def _set_bandnames(self, band_names=None):
             band_names = [band_names]
         self._band_names = list(band_names)
 
+    def subimage(self, bands):
+        if isinstance(bands, str):
+            bands = [bands]
+
+        missing_bands = set(bands) - set(self.band_names)
+        if missing_bands:
+            raise GeoRaster2Error('requested bands %s that are not found in raster' % missing_bands)
+
+        bands_indices = [self.band_names.index(band) for band in bands]
+        subimage = self.image[bands_indices, :, :]
+        return subimage
```

Comment:
we have the same problem with `raster.image` why there it isn't a problem and here it is, the idea behind this method is to do the same as `.image` but slice by bands 

I don' want to copy, since that is the whole point of optimzing, and in the context of what I am doing it okay since there is no direct equality